src/cgame/cg_rocket_draw.cpp

```diff
@@ -2442,6 +2442,35 @@ class NumSpawnsElement : public TextHudElement
 	int spawns_;
 };
 
+class PlayersCountElement : public TextHudElement
+{
+public:
+	PlayersCountElement( const Rml::String& tag ) :
+			TextHudElement( tag, ELEMENT_GAME )
+			{}
+
+	void DoOnUpdate() override
+	{
+		//there is probably a better way to do that, maybe reusing value
+		//cached in some struct?
+		std::string teamname = GetAttribute< Rml::String >( "team", "" );
+
+		team_t team = BG_PlayableTeamFromString( teamname.c_str() );
+		size_t count = 0;
+
+		for ( int i = 0; i < cg.numScores; ++i )
```

Comment:
For the cgame, probably not...

The only option might be `rocketInfo.data.playerCount[ team ]`

But this isn't guaranteed to be updated.

Also, this will be run for every element, which is bad. We should run this once, and use the value for all elements.

Reply:
wdym "for every element"? This code is not run only for the element I wrote a class for?

Reply:
I guess he means for every `TextHudElement`, and this is just a call to move the for loop outside of that RML c++ code and just use there a variable that is computed elsewhere and once for all per frame.